src/test/java/org/generationcp/middleware/service/impl/inventory/LotServiceImplIntegrationTest.java

```diff
@@ -164,4 +226,14 @@ private void resolveStorageLocation() {
 		this.storageLocationId = this.daoFactory.getLocationDAO().getDefaultLocationByType(id).getLocid();
 	}
 
+	private void assertTransaction(Transaction actualTransaction, TransactionType type, TransactionStatus status,
```

Comment:
to be consistent I will add final to the params @lik-83

Reply:
Done!